docker/sirius/sirius-config.yaml

```diff
@@ -1,2 +1,63 @@
 plugin: openapi
 specFile: openapi.yml
+
+resources:
+  - path: "/api/v1/digital-lpas/M-XYXY-YAGA-35G3"
+    method: get
+    response:
+      statusCode: 200
+      scriptFile: useDonorJsonFile.groovy
+
+  - path: "/api/v1/digital-lpas/M-XYXY-YAGA-35G4"
+    method: get
+    response:
+      statusCode: 200
+      scriptFile: useDonorJsonFileNoDecision.groovy
+
+  - path: "/api/v1/digital-lpas/M-XYXY-YAGA-35G0"
+    method: get
+    response:
+      statusCode: 200
+      scriptFile: useDonorJsonFileStop.groovy
+
+  - path: "/api/v1/digital-lpas/M-XYXY-YAGA-0000"
+    method: get
+    response:
+      statusCode: 200
+      scriptFile: useCertificateProviderJsonFile.groovy
+
+  - path: "/api/v1/digital-lpas/M-XYXY-YAGA-0001"
+    method: get
+    response:
+      statusCode: 200
+      scriptFile: useCertificateProviderJsonFileNoDecision.groovy
+
+  - path: "/api/v1/digital-lpas/M-XYXY-YAGA-0002"
+    method: get
+    response:
+      statusCode: 200
+      scriptFile: useCertificateProviderJsonFileStop.groovy
+
+  - path: "/api/v1/users/current"
+    method: get
+    response:
+      statusCode: 200
+      scriptFile: useUserJsonFile.groovy
+
+  - path: "/api/v1/identity-check"
+    method: post
+    response:
+      statusCode: 204
+      scriptFile: useIdentityCheckJsonFile.groovy
+
+  - path: "/api/v1/lpas/M-XYXY-YAGA-35G3/documents"
+    method: post
+    response:
+      statusCode: 201
+      scriptFile: usePdfResponseJsonFile.groovy
+
+  - path: "/api/v1/lpas/M-XYXY-YAGA-0000/documents"
+    method: post
+    response:
+      statusCode: 201
+      scriptFile: usePdfResponseJsonFile.groovy
```

Comment:
I don't think any of these should be needed since it will already return like this based on the OpenAPI spec?